Gems/Maestro/Code/Source/Cinematics/AnimComponentNode.h

```diff
@@ -150,16 +150,16 @@ class CAnimComponentNode
         }
         BehaviorPropertyInfo(const BehaviorPropertyInfo& other)
         {
-            m_displayName = other.m_displayName;
-            m_animNodeParamInfo.paramType = other.m_displayName;
-            m_animNodeParamInfo.name = &m_displayName[0];
+            m_displayName = AZStd::move(other.m_displayName);
```

Comment:
update: I don't think this should be a `move` as this is the copy constructor, `other.m_displayName` should be left as-is and not be moved from